src/runtime/virtcontainers/sandbox.go

```diff
@@ -2576,11 +2576,11 @@ func (s *Sandbox) GetPatchedOCISpec() *specs.Spec {
 		return nil
 	}
 
-	// get the container associated with the PodSandbox annotation. In Kubernetes, this
-	// represents the pause container. In Docker, this is the container.
-	// On Linux, we derive the group path from this container.
+	// Get the container associated with the PodSandbox annotation.
+	// In Kubernetes, this represents the pause container. In Docker, this is the container.
```

Comment:
It seems that "In Docker"  is not appropriate today. Should we make it more generic?

Reply:
Yes, I changed it to "CRI-compatible runtime, such as Containerd"